Userland/Utilities/sql.cpp

```diff
@@ -156,6 +169,39 @@ class SQLRepl {
     AK::RefPtr<SQL::SQLClient> m_sql_client { nullptr };
     int m_connection_id { 0 };
     Core::EventLoop m_loop;
+    FILE* m_input_file { nullptr };
+    bool m_quit_when_files_read { false };
+    Vector<String> m_input_file_chain {};
+
+    Optional<String> get_line()
+    {
+        if (!m_input_file && !m_input_file_chain.is_empty()) {
+            auto file_name = m_input_file_chain.take_first();
+            m_input_file = fopen(file_name.characters(), "r");
```

Comment:
Instead of `FILE` / `fopen`, we should use `Core::File` (which has a `read_line` method)

Reply:
I completely forgot about ``Core::File``. 🤦